src/bundle/Resources/config/override/ezpublish.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
```

Comment:
`content_fields.html.twig` and `fielddefinition_settings.html.twig` are really the ones this is a problem for. All of the other templates are clearly meant to be overridden.

Should we just exclude those ? Another option is to make sure that any override inherits the standard template...

Reply:
In reply to out-of-thread https://github.com/ezsystems/ezplatform-standard-design/pull/2#issuecomment-392535626

> I don't see a usecase where these default templates should be part of the standard design.
Then let's say that field templates aren't part of the standard design. What do you think @alongosz ?

About the other templates, I do: the admin. It must override most of these.

Reply:
Well, admin can define it's own base templates, for `admin` scope.

Reply:
That's not how we plan to do it.

I perfectly understand and accept the errors that can be caused by having field templates in the standard design.

The fact that you don't see the point of having the other ones in it doesn't mean it is a problem.

Reply:
Isn't the main purpose of this PR to let people override templates, even if they can break things? I thought this was our main goal behind it. I don't see a problem if someone will put template with the same name and break it. It is developer responsibility after all.

Reply:
I tend to disagree, because the fact is that we did run into the same issues a number of times when using ezdesign engine because we relied on template location and name for unrelated contexts, like frontend and (ng)admin and they are a hell to debug and fix.

Reply:
@lserwatka Field (def) templates already have their own override system, because you can inject them into `field_settings` with some priority, grouped by the scope. Introducing design engine into the process, brings ambiguity into the process.

Besides, I would definitelly not expect for my site to break because I placed a template into *my own* bundle at a generic location.

Reply:
We need to distinguish field templates and the other ones (`*/view/*`) @lserwatka. @emodric raises a fair point with those.

There is no realistic case where you'd want to override them, and we have [another mechanism](https://doc.ezplatform.com/en/2.1/guide/twig_functions_reference/#global-override) dedicated to them.

Reply:
But then we have a mess. Two or three different mechanism to override templates. Nobody will understand this. It is unnecessary complexity and inconsistency.

Reply:
It depends how you look at it. How should 3rd party bundles provide their own field type templates that can easily be overriden by other 3rd party bundles?

Design engine complicates that usecase a lot as it wouldn't be possible to do the overrides automatically and would depend on bundle order in app kernel.